src/ucp/rndv/proto_rndv.inl

```diff
@@ -265,6 +270,48 @@ ucp_proto_rndv_bulk_max_payload(ucp_request_t *req,
     return max_payload;
 }
 
+/**
+ * Return the fragment size to ensure alignment for buffer of main data
+ * portion. Either return max_payload when the buffer is initially aligned,
+ * or alignment fragment already sent.
+ */
+static UCS_F_ALWAYS_INLINE size_t
+ucp_proto_rndv_bulk_max_payload_align(ucp_request_t *req,
+                                      const ucp_proto_rndv_bulk_priv_t *rpriv,
+                                      const ucp_proto_multi_lane_priv_t *lpriv,
+                                      ucp_lane_index_t *lane_shift)
+{
+    size_t total_offset = ucp_proto_rndv_request_total_offset(req);
+    size_t align_thresh = rpriv->mpriv.align_thresh;
+    size_t align        = lpriv->opt_align;
+    size_t max_payload, align_size;
+    unsigned buffer_padding;
+    void *buffer;
+
+    ucs_assertv(align != 0, "align=%zu", align);
+    ucs_assertv(req->send.state.dt_iter.dt_class == UCP_DATATYPE_CONTIG,
+                "dt_class=%d (%s)", req->send.state.dt_iter.dt_class,
+                ucp_datatype_class_names[req->send.state.dt_iter.dt_class]);
+
+    *lane_shift = 1;
+
```

Comment:
i think this is not needed since lane_shift would be already initialized by default to 1 by the multi-send function